### PR TITLE
fix: language selector vertical align

### PR DESCRIPTION
**Pull request by araujogui**
## Description

Fix LanguageSelector vertical align.

### Check List

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run `npx turbo lint` to ensure the code follows the style guide. And run `npx turbo lint:fix` to fix the style errors if necessary.
- [x] I have run `npx turbo format` to ensure the code follows the style guide.
- [x] I have run `npx turbo test` to check if all tests are passing, and/or `npx turbo test:snapshot` to update snapshots if I created and/or updated React Components.
- [x] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

**Review by ovflowd — APPROVED** [review]
LGTM! Nice fix 🙇

File: components/Common/LanguageSelector/index.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const LanguageSelector = () => {
   });
 
   return (
-    <div className={styles.container} ref={ref}>
+    <>
```
Comment (bmuenzenmeyer) [inline components/Common/LanguageSelector/index.tsx:44]: I don't think this fragment is necessary anymore, right?

**Review by bmuenzenmeyer — APPROVED** [review]
Looks good to me.

One small nit in the comments.

## Validation

Also ✅ on the header alignment.

image

File: components/Common/LanguageSelector/index.tsx (hunk 1 above)
Comment (ovflowd) [reply components/Common/LanguageSelector/index.tsx:44]: It is, as there are two elements in the same level here

Comment (bmuenzenmeyer) [reply components/Common/LanguageSelector/index.tsx:44]: Oh.. Lol the ui was collapsing the full content to make it look like a single element. My bad

**Comment by shanpriyan** [comment]
@araujogui what was the issue 🤔 , I can see from PR that the `useClickOutside` hook usage on DropDown has been fixed.

**Comment by ovflowd** [comment]
@shanpriyan actually the bug this PR fixes is the vertical alignment of the language selector button.

But we also moved the ref handling to the Dropdown itself.

As it makes more sense

**Comment by shanpriyan** [comment]
> @shanpriyan actually the bug this PR fixes is the vertical alignment of the language selector button.
> 
> But we also moved the ref handling to the Dropdown itself.
> 
> As it makes more sense

Got it 👍🏻

**Review by shanpriyan — APPROVED** [review]
LGTM! 🚀

**Pull request merged**